src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -92,55 +124,55 @@ class CompanyStep extends React.Component {
     }
 
     /**
-     * @returns {Boolean}
+     * Clear the error associated to inputKey if found and store the inputKey new value in the state.
+     *
+     * @param {String} inputKey
+     * @param {String} value
      */
-    validate() {
-        if (!this.state.password.trim()) {
-            showBankAccountFormValidationError(this.props.translate('common.passwordCannotBeBlank'));
-            return false;
+    clearErrorAndSetValue(inputKey, value) {
+        this.setValue({[inputKey]: value});
+        const errors = this.getErrors();
+        if (!errors[inputKey]) {
+            // No error found for this inputKey
+            return;
         }
 
-        if (!isValidAddress(this.state.addressStreet)) {
-            showBankAccountFormValidationError(this.props.translate('bankAccount.error.addressStreet'));
-            return false;
-        }
+        // Clear the existing error for this inputKey
+        const newErrors = {...errors};
+        delete newErrors[inputKey];
+        setBankAccountFormValidationErrors(newErrors);
+    }
 
-        if (this.state.addressState === '') {
-            showBankAccountFormValidationError(this.props.translate('bankAccount.error.addressState'));
-            return false;
+    /**
+     * @returns {Boolean}
+     */
+    validate() {
+        const errors = {};
+        if (!isValidAddress(this.state.addressStreet)) {
+            errors.addressStreet = true;
         }
-
         if (!isValidZipCode(this.state.addressZipCode)) {
-            showBankAccountFormValidationError(this.props.translate('bankAccount.error.zipCode'));
-            return false;
+            errors.addressZipCode = true;
         }
-
         if (!Str.isValidURL(this.state.website)) {
-            showBankAccountFormValidationError(this.props.translate('bankAccount.error.website'));
-            return false;
+            errors.website = true;
         }
-
         if (!/[0-9]{9}/.test(this.state.companyTaxID)) {
-            showBankAccountFormValidationError(this.props.translate('bankAccount.error.taxID'));
-            return false;
+            errors.companyTaxID = true;
         }
-
         if (!isValidDate(this.state.incorporationDate)) {
-            showBankAccountFormValidationError(this.props.translate('bankAccount.error.incorporationDate'));
-            return false;
+            errors.incorporationDate = true;
         }
-
         if (!isValidIndustryCode(this.state.industryCode)) {
-            showBankAccountFormValidationError(this.props.translate('bankAccount.error.industryCode'));
-            return false;
+            errors.industryCode = true;
         }
-
```

Comment:
NAB, removing these new lines seems unnecessary. I like them personally since things are less visually compressed, but that's just my opinion and there are no style rules about this as far as I know.

Reply:
Got mixed up with the [php guidelines](https://github.com/Expensify/Style-Guides/blob/master/php.md#blank-lines). I assumed that a similar rule would apply here... reverting!